lib/cmd/compile/scripts.js

```diff
@@ -143,14 +143,15 @@ function getModuleId(file, legacyFiles) {
   } else if (isLegacyFile) {
     return `${path.parse(file).name}.legacy`;
   } else if (_.includes(file, path.join(process.cwd(), 'components'))) {
-    fileTypes.forEach((type) => {
-      if (_.endsWith(file, `${type}.js`)) {
-        return `${name}.${type}`;
+    for (let x = 0; x < fileTypes.length; x++ ) {
```

Comment:
You know, I don't like spacing 😬 
```suggestion
    for (let x = 0; x < fileTypes.length; x++) {
```

Reply:
That was just a test to see if you were paying attention.  You passed